src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PropsPagination = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}: PropsPagination) => {
+  const items = getNumbers(1, total);
+  const totalPages = Math.ceil(items.length / perPage);
+  const totalPagesArr = getNumbers(1, totalPages);
```

Comment:
You can calculate `totalPagesArr` directly without using `getNumbers` since you already have the `totalPages` value. Consider using a loop to push page numbers into an array if needed.

Reply:
My code would become mess without these extra variables. Why do I need util helper function getNumbers if I am gonna replace it with for loop that does exact same thing but takes more space in code and look uglier ?